googleapis_auth/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 ## 1.6.1-wip
 
 - Require `sdk: ^3.5.0`
+- Remove `allowInterop` calls for callbacks passed to `CodeClientConfig`,
+  `TokenClientConfig`, and `revoke` as they only accept Dart functions.
+- Fix the type of the callback passed to `revoke`.
```

Comment:
```suggestion
```

Users don't care. 😄

Reply:
That's what I thought :D

I'll keep the `dart:js_interop` migration bit though.